components/registry-facade/pkg/registry/registry.go

```diff
@@ -420,7 +420,7 @@ type dispatchFunc func(ctx context.Context, r *http.Request) http.Handler
 func dispatcher(d dispatchFunc) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		fc, _ := httputil.DumpRequest(r, false)
-		fmt.Fprint(os.Stderr, string(fc))
+		log.Info(string(fc))
```

Comment:
Why not just change the writer? `fmt.Fprint(os.Stout, string(fc))`

Reply:
Or a more "traditional" structured log statement: `log.WithField("req", string(fc)).Debug("dispatching request")`